x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -2895,62 +2895,8 @@
     "indexPatternManagement.actions.createButton": "フィールドを作成",
     "indexPatternManagement.actions.deleteButton": "削除",
     "indexPatternManagement.actions.saveButton": "フィールドを保存",
-    "indexPatternManagement.aliasLabel": "エイリアス",
     "indexPatternManagement.createHeader": "スクリプトフィールドを作成",
-    "indexPatternManagement.createIndexPattern.betaLabel": "ベータ",
```

Comment:
I am not 100% sure how this is supposed to be, but since we've didn't removed these labels, but just renamed the keys, then shouldn't we also rename keys here instead of just removing the translations?

Reply:
I've defaulted to removing keys because contextual changes might change translations even if western languages would remain the same. Certainly needs a pass from a translator.